.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,89 @@
+name: code coverage
```

Comment:
Yay forge coverage! Guessing you also pulled from v4? It definitely needs to be updated.. We should brainstorm how to do this better. I'd love for it to be not a bot, but something that you can see a diff with

Reply:
(This can definitely be a future TODO btw) I just wanted to call it out

Reply:
yes, seems like a bit more effort we can put in at a later point in time, definitely in favor of this idea